packages/cozy-sharing/src/components/CozyPassFingerprintDialogContent.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import snarkdown from 'snarkdown'
+
+import cx from 'classnames'
+import styles from './CozyPassFingerprintDialogContent.styl'
+
+/**
+ * CozyPassFingerprintDialogContent can be use as a content for ShareModal when
+ * using 2 steps confirmation
+ *
+ * This should be the case when ShareModal is called from `cozy-pass-web`
+ */
+export const CozyPassFingerprintDialogContent = ({
+  recipientConfirmationData
+}) => {
+  const { t } = useI18n()
+
+  const instruction1 = snarkdown(
+    t(`Organizations.fingerprint.instruction`, {
+      userName: recipientConfirmationData.name,
+      userEmail: recipientConfirmationData.email
+    })
+  )
+
+  const instruction2 = snarkdown(t(`Organizations.fingerprint.instruction2`))
+
+  return (
+    <div className={cx(styles['share-modal-content'])}>
+      <p dangerouslySetInnerHTML={{ __html: instruction1 }}></p>
+      <Paper
+        className={cx(styles['cozy-pass-fingerprint-modal__fingerprint'])}
```

Comment:
I think you can use utility class for this kind of stylus. If we can avoid the creation of stylus file, let's do it :)

Reply:
Do you mean something like `u-mr-half`? I just found documentation on https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html but where are those declared? I don't find any declaration when looking for `u-mr-half` in githug organization nor in material-ui.

Reply:
https://github.com/cozy/cozy-ui/blob/67485ca3a6db28dd4ba64d2c91188c21b00034ce/stylus/utilities/spaces.styl 
https://unpkg.com/browse/cozy-ui@51.5.0/dist/cozy-ui.utils.min.css

Reply:
Seems like sementic colors are not implemented in utility classes so I cannot replace `color var(--secondaryColor)`. Should I inline this style instead? Or should I keep the stylus?

Reply:
This is a good question, but here you should not use `<p>`. You should use the correct variant of `Typography` component and with this component you can use the color props. No?

Reply:
Not sure understand you, the current code is already based on `Typography` with `body1` variant. The `<p>` is resulting from this.

Or maybe you are talking about the `<p>` containing `instruction1` ? If yes I can convert them. But the `p` targeted by `color var(--secondaryColor)` is the one from the current `Typography` inside of the `Paper`. I just don't know how to reference that semantic variable from the JS code without inlining css.